src/snap.ts

```diff
@@ -1,13 +1,22 @@
 import {Wallet} from "./interfaces";
+import {getKeysFromSeed} from "./util";
 
 declare let wallet: Wallet;
 
 wallet.registerRpcMessageHandler(async (originString, requestObject) => {
   switch (requestObject.method) {
     case 'hello':
+      const appKey = await wallet.getAppKey();
+      const keypair = getKeysFromSeed(appKey);
```

Comment:
should be generated on connect

Reply:
@mpetrunic I would leave for now invocation on button click event because I am still waiting for some more information on how to invoke this code on connect event. We can refactor this in the near feature.

Reply:
sure